rspace/src/main/scala/coop/rchain/rspace/ISpace.scala

```diff
@@ -13,9 +13,10 @@ import scala.collection.immutable.Seq
   * @tparam C a type representing a channel
   * @tparam P a type representing a pattern
   * @tparam A a type representing an arbitrary piece of data
+  * @tparam A a type representing a match result
```

Comment:
`@tparam R`

Reply:
Can match result be different type that the data?